src/annotations/components/annotationForm/AnnotationForm.tsx

```diff
@@ -83,64 +103,148 @@ export const AnnotationForm: FC<Props> = (props: Props) => {
   }
 
   const handleKeyboardSubmit = () => {
-    props.onSubmit({message, startTime, endTime})
+    props.onSubmit({
+      summary,
+      startTime,
+      endTime,
+      type: annotationType,
+      id: props.id,
```

Comment:
understood; better to clear up confusion that have the code be elegant; because if it's confusing => the next refactor more likely to introduce a bug.

will switch to annotationId